spec/support/capybara.rb

```diff
@@ -1,7 +1,22 @@
-require "capybara/poltergeist"
+require "selenium/webdriver"
+
+Capybara.register_driver :chrome do |app|
```

Comment:
Could we skip this b/c of https://github.com/teamcapybara/capybara/blob/master/lib/capybara.rb#L485-L487 ?

Reply:
I think that this configuration should be left, because in this way we clearly show which drivers are used in our project. Also writing `driver: :chrome` when debugging much faster and easier to remember than` driver: :selenium_chrome`

Reply:
In Rails we follow "Convention over Configuration" and this case it will be better to stick with defined defaults in Capybara.